### ISSUE TITLE
Got wrong day of the week when not generating indates

### ISSUE THREAD

**Issue by zjdavid** [patchthecode/JTAppleCalendar#440]
I'm currently using version 7.0.4.
I found that the first date of the month became Sunday when I used `.off` for parameter `generateInDates`. 

![img_0001](https://user-images.githubusercontent.com/16710181/26908868-6e4be420-4bcb-11e7-94a1-b39c4304f99b.PNG)

I made a sample project here:

[testProject.zip](https://github.com/patchthecode/JTAppleCalendar/files/1059743/testProject.zip)

**Comment by patchthecode** [comment]
This is expected.

Think about this, if you do not generate outdates, this means that the `1st` will always be on the extreme left right.

Therefore, if the first is always on the extreme left, then under what column should the first be considering that you are the one that maintains the [Su | Mo |Tu | We| Th | Fr | Sa] header?

If this causes a design problem for you then let me know how you want you calendar to look, and then I will tell you what setup you should put in your `configureCalendar` function

**Comment by patchthecode** [comment]
Will close this, but if you have more questions, feel free to ask ok.

**Comment by ahmed-khalil** [comment]
Would be possible to have the same location for the first day of the month however without the indates? As if the indates are hidden?

**Comment by patchthecode** [comment]
@ahmed-khalil could you rephrase that? I still do not understand the question. thanks
